api/src/main/java/io/strimzi/api/kafka/model/KafkaBridge.java

```diff
@@ -39,14 +40,22 @@
                 ),
                 group = KafkaBridge.RESOURCE_GROUP,
                 scope = KafkaBridge.SCOPE,
-                version = KafkaBridge.V1ALPHA1,
+                version = KafkaBridge.V1BETA1,
```

Comment:
Wait ... are you adding KafkaBridge v1beta1? Why? Is it really needed to add the status? It does IMHO not break the compatibility. This has upgrading impact, so we should avoid it unless it is needed.

Reply:
As far as I can remember I got some error messages about `alpha` not supporting status subresource, but I can doublecheck.

Reply:
I think we need to investigate this. The v1alpha1 -> v1beta1 change for the other resources was not because of the status but the other changes. If it is needed to upgrade you will need to take care of documenting it in release notes etc.

Reply:
You were right. Reverted to alpha.